travis/test_server.py

```diff
@@ -142,8 +143,11 @@ def get_addons_path(travis_home, travis_build_dir, server_path):
     return addons_path
 
 
-def get_server_script(odoo_version):
-    return 'odoo-bin' if float(odoo_version) >= 10 else 'openerp-server'
+def get_server_script(server_path):
+    if os.path.isfile(os.path.join(server_path, 'odoo-bin')):
```

Comment:
I told you: Review the issue 3 times before of create a change...
You will see that the change was approved to fix it with `try-except` and not with `if isfile` 😞